themes/_core/js/address.js

```diff
@@ -14,7 +14,7 @@ function handleCountryChange(selectors) {
       id_address: $(`${selectors.address} form`).data('id-address'),
     };
     const getFormViewUrl = $(`${selectors.address} form`).data('refresh-url');
-    const formFieldsSelector = `${selectors.address} input`;
+    const formFieldsSelector = `${selectors.address} :input`;
```

Comment:
Because `:input` is a jQuery extension and not part of the CSS specification, queries using `:input` cannot take advantage of the performance boost provided by the native DOM `querySelectorAll()` method. To achieve the best performance when using `:input` to select elements, first select the elements using a pure CSS selector, then use `.filter(":input")`.

From https://api.jquery.com/input-selector/

I'm sorry, it's not the right way to use this selector :/

Reply:
Agreed with @PierreRambaud 

Sorry we cant accept this contribution 😞